internal/openshift/dashboards/metrics-dashboard.json

```diff
@@ -0,0 +1,1175 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "$$hashKey": "object:517",
+        "builtIn": 1,
+        "datasource": "-- Grafana --",
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "type": "dashboard"
+      }
+    ]
+  },
+  "description": "Provides information about the status of the OpenTelemetry Collector",
```

Comment:
this is a nit: but can we format this json file, this is pretty hard to read right now.

Reply:
Done